src/clojisr/v1/require.clj

```diff
@@ -69,45 +76,64 @@
         (seq opt) (list ['& {:keys opt}])
         :else '([])))))
 
-(defn r-symbol->clj-symbol [r-symbol r-object]
+(defn- safe-help [r-object]
+  (try 
+    (help/help r-object)
+    (catch Exception e "")))
+
+(defn r-symbol->clj-symbol [ r-symbol r-object] 
+  
   (if-let [arglists (r-object->arglists r-object)]
     (vary-meta r-symbol assoc :arglists arglists)
     r-symbol))
 
+
+(defn- assoc-doc-to-meta! [ns-symbol r-symbol r-object]
+  (alter-meta!
+   (get (ns-publics ns-symbol) r-symbol)
+   assoc :doc (safe-help r-object)))
+
 (defn add-to-ns [ns-symbol r-symbol r-object]
   (intern ns-symbol
           (r-symbol->clj-symbol r-symbol r-object)
-          r-object))
+          r-object)
+  ns-symbol)
```

Comment:
Why do we need this (ie. returned symbol)?

Reply:
Looks like mistake.  Will fix in next commit.